app/src/main/java/app/musikus/ui/theme/Theme.kt

```diff
@@ -74,47 +73,37 @@ fun MusikusTheme(
         }
     }
 
-    // source: https://gist.github.com/Khazbs/1f1f1b5c05f45dbfa465f249b1e20506
-    // and https://stackoverflow.com/questions/65610216/how-to-change-statusbar-color-in-jetpack-compose
-    val view = LocalView.current
-    if (!view.isInEditMode) {
-        SideEffect {
-            val window = (view.context as Activity).window
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
-                window.setDecorFitsSystemWindows(false)
-
-                // background extends behind status and nav bar (even 3 button nav bar)
-                window.setFlags(
-                    WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS,
-                    WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS
-                )
-            } else {
-                WindowCompat.setDecorFitsSystemWindows(window, false)
-
-                // status bar icons can adapt to light/dark theme,
-                // so we can make the status bar transparent
-                window.statusBarColor = Color.Transparent.toArgb()
-
-                // navigation bar icons can only adapt to light theme for API >= 26 (O),
-                // otherwise we need to manually set the nav bar color to something dark
-                window.navigationBarColor = if(
-                    Build.VERSION.SDK_INT >= Build.VERSION_CODES.O || useDarkTheme
-                ) {
-                    Color.Transparent.toArgb()
-                } else {
-                    LightColorScheme.inverseSurface.toArgb()
-                }
-            }
+    // enable Edge-to-Edge drawing with new API
+    val context = LocalContext.current as ComponentActivity
+    DisposableEffect(key1 = theme) {
```

Comment:
Why are you using a DisposableEffect when you are not using the onDispose function? I would suggest using a LaunchedEffect instead

Reply:
Hm, you may be right.... I just orientated myself on [the example](https://medium.com/@shubhgajjar2004/different-ways-to-change-the-color-of-status-bar-and-navigation-bar-in-android-jetpack-compose-edd1c5542257). But maybe a `LaunchedEffect` will do it's job just as good...